server/src/main/java/io/crate/expression/symbol/Symbols.java

```diff
@@ -265,12 +265,6 @@ public Void visitReference(Reference ref, Void context) {
                     if (t instanceof Reference tRef && tRef.column().equals(root)) {
                         consumer.accept(t);
                         break;
-                    } else if (ref instanceof VoidReference
-                               && t instanceof ScopedSymbol tScopedSymbol
-                               && tScopedSymbol.relation().equals(ref.ident().tableIdent())
-                               && tScopedSymbol.column().equals(root)) {
-                        consumer.accept(t);
-                        break;
```

Comment:
And here, why shouldn't VoidReferences be visited anymore? I think we still want to process them on e.g. output pruning.

Reply:
Reverted, same as https://github.com/crate/crate/pull/17133#discussion_r1935150098